internal/acctest/acctest.go

```diff
@@ -1925,3 +1926,30 @@ func CheckResourceAttrGreaterThanValue(n, key, value string) resource.TestCheckF
 
 	}
 }
+
+func imageBuilderPlatformIsValid(platform string) bool {
+	for _, p := range imagebuilder.Platform_Values() {
+		if p == platform {
+			return true
+		}
+	}
+	return false
+}
+
+func ImageBuilderECRImageFromEnv(t *testing.T, platform string) string {
+	if !imageBuilderPlatformIsValid(platform) {
+		t.Skip(fmt.Sprintf("Platform %q is not valid for Image Builder. Valid values are %v.", platform, imagebuilder.Platform_Values()))
+	}
+
+	ecrImage := os.Getenv(fmt.Sprintf("IMAGE_BUILDER_ECR_IMAGE_%s", strings.ToUpper(platform)))
+
+	if ecrImage == "" {
+		t.Skip(
+			fmt.Sprintf("Environment variable IMAGE_BUILDER_ECR_IMAGE_%s is not set. " +
+				"For AWS EC2 Image Builder tests using an ECR Image" +
+				"as the Parent Image the ECR Image must be accessible for" +
+				"testing.", string.ToUpper(platform)))
```

Comment:
```suggestion
				"testing.", strings.ToUpper(platform)))
```
Apologies :disappointed:  Plus you'll need to run `make fmt` - evidently trying to hack on code in the GitHub UI isn't working well for me today!